dapps/tests/app/app/js/index.js

```diff
@@ -1,195 +1,159 @@
+/*global web3*/
+import EmbarkJS from 'Embark/EmbarkJS';
+import {SimpleStorage, Test, SimpleStorageTest} from '../../embarkArtifacts/contracts';
+
+window.SimpleStorageTest = SimpleStorageTest;
 
-// import {SimpleStorage} from '../../embarkArtifacts/contracts';
 import $ from 'jquery';
-import SimpleStorage from '../../embarkArtifacts/contracts/SimpleStorage.js';
-// import web3 from '../../embarkArtifacts/contracts/web3_init';
-// window.web3 = web3;
-window.SimpleStorage = SimpleStorage;
-import EmbarkJS from '../../embarkArtifacts/embarkjs.js';
+import 'bootstrap/dist/css/bootstrap.css';
+import 'bootstrap/dist/css/bootstrap-theme.css';
+import 'bootstrap/dist/js/bootstrap.min.js';
+
 window.EmbarkJS = EmbarkJS;
+window.SimpleStorage = SimpleStorage;
+window.Test = Test;
+
+var addToLog = function(id, txt) {
+  $(id + " .logs").append("<br>" + txt);
+};
+
+// ===========================
+// Blockchain example
+// ===========================
+$(document).ready(function() {
+  EmbarkJS.onReady((err) => {
+    if (err) {
+      console.error(err);
+    }
+  });
+
+  console.log([1,2,3].map(v => v + 1));
 
-$("#blockchain button.set").click(function() {
-  var value = parseInt($("#blockchain input.text").val(), 10);
+  $("#blockchain button.set").click(function() {
+    var value = parseInt($("#blockchain input.text").val(), 10);
+
+    SimpleStorage.methods.set(value).send({from: web3.eth.defaultAccount, gas: 5300000});
+    addToLog("#blockchain", "SimpleStorage.methods.set(value).send({from: web3.eth.defaultAccount, gas: 5300000})");
+  });
+
+  $("#blockchain button.get").click(function() {
+    SimpleStorage.methods.get().call(function(err, value) {
+      $("#blockchain .value").html(value);
+    });
+    addToLog("#blockchain", "SimpleStorage.methods.get(console.log)");
+  });
 
-  SimpleStorage.methods.set(value).send({from: web3.eth.defaultAccount, gas: 5300000});
 });
 
-// TODO check why from not set
-$("#blockchain button.get").click(function() {
-  SimpleStorage.methods.get().call({from: web3.eth.defaultAccount}, function(err, value) {
-    console.log('GET', err, value);
-    $("#blockchain .value").html(value);
+// ===========================
+// Storage (IPFS) example
+// ===========================
+$(document).ready(function() {
+  // automatic set if config/storage.json has "enabled": true and "provider": "ipfs"
```

Comment:
Can we get rid of the commented code if we don't need it and it might not be uncommented later?